### ISSUE TITLE
Test transformer with GPU

### ISSUE THREAD

**Issue by Humorloos** [Humorloos/web_mining#46]
Prepare easily runnable file
run overnight

**Comment by Humorloos** [comment]
Questions to resolve:
- How do we decrease memory consumption?
  - Use an adapter?
  - No fine-tuning?
  - freeze some base-model layers?
  - Use different base model?
  - Avoid larger batch sizes?

**Comment by Humorloos** [comment]
Steps for setup:
- if not running in pycharm, move train, tune, and evaluation script to src directory
- setup wandb logger by entering api key
- install correct versions of python packages: 
  - `adapter-transformers==2.1.2?`
  - `ConfigSpace==0.5.0`
  - ray-tune (via conda)
  - adapter-transformers==2.1.2
  - maybe others??

**Comment by Humorloos** [comment]
#### Options for GPUs:
- dws cluster (access requested, wait for credentials)
- Bw-uni cluster chia
  - [Category:BwUniCluster 2.0 - bwHPC Wiki](https://wiki.bwhpc.de/e/Category:BwUniCluster_2.0)

#### hyperparameter optimization options to reach convergence:
- Larger batch size
- smaller learning rate (1e-6)
- turn off early stopping
